js/pages/backoffice/toolbox.js

```diff
@@ -201,7 +201,10 @@ CombodoModal._InstantiateModal = function(oModalElem, oOptions) {
 		width: 'auto',
 		height: 'auto',
 		modal: oOptions.extra_options.modal ?? true,
+		close: oOptions.extra_options.callback_on_modal_close,
 		autoOpen: oOptions.auto_open,
+		title: oOptions.title,
+		buttons: CombodoModal.ConvertButtonDefinition(oOptions.buttons)
```

Comment:
```suggestion
		buttons: this.ConvertButtonDefinition(oOptions.buttons)
```

Do you think using "this" instead of a full class name would be more explicit in understanding that it actually refers to something of the same object?